scale/scheduler/database/updater.py

```diff
@@ -118,6 +106,7 @@ def _perform_batch_field_init(self):
         self._total_batch = Batch.objects.all().count()
         logger.info('Found %d batches that need to be done', self._total_batch)
 
+    # TODO: remove/fix with v5 removal
```

Comment:
Comment can be removed?

Reply:
Are we able to remove these updater methods? As far as I understand they are supposed to be one time conversions. At this point would it be safe to assume these conversions have all been applied and no new conversions are needed?

Reply:
Realized these were only marked as TODO: remove/fix in our v5 remove branch. Since the methods have been updated, I can remove the comments.